plugins/inputs/t128_peer_path/t128_peer_path.go

```diff
@@ -74,6 +76,17 @@ func (*T128PeerPath) Description() string {
 // Init sets up the input to be ready for action
 func (plugin *T128PeerPath) Init() error {
 
+	var query string
+	if plugin.ExcludeHostname {
+		if _, found := peerPathTags["adjacentHostname"]; found {
+			delete(peerPathTags, "adjacentHostname")
```

Comment:
You don't need to check for existence before `delete`ing. Nothing bad happens if it's missing.